app/views/skins/_wizard_form.html.erb

```diff
@@ -1,59 +1,110 @@
+      </p>
     </dd>
-  
+
     <dt>
-      <%= f.label :base_em, ts('Percent of base font size (%)') %> 
-      <%= link_to_help('skins-wizard-font-size') %>
+      <%= f.label :margin, ts('Work margin width') %>
     </dt>
-    <dd><%= f.text_field :base_em %></dd>
-    
+    <dd>
+      <%= f.text_field :margin, "aria-describedby" => "margin-field-notes" %>
+      <p class="footnote" id="margin-field-notes">
+        <%= ts("Numbers only, treated as a percentage of the page width.") %>
+      </p>
+    </dd>
+
     <dt>
       <%= f.label :paragraph_margin, ts('Vertical gap between paragraphs') %> 
       <%= link_to_help('skins-wizard-vertical-gap') %>
     </dt>
-    <dd><%= f.text_field :paragraph_margin %></dd>
-    
-    <dt><%= f.label :headercolor, ts('Header color') %> 
-    <%= link_to_help('skins-wizard-header-color') %>
+    <dd>
+      <%= f.text_field :paragraph_margin %>
+      <p class="footnote" id="paragraph-margin-field-notes">
+        <%= ts("Numbers only, treated as a multipler of the paragraph font size. Default: <code>1.286</code>").html_safe %>
+      </p>
+    </dd>
+  </dl>
+</fieldset>
+
+<fieldset>
+  <legend><%= ts("Colors") %></legend>
+  <p class="notes">
+    <%= ts('You may wish to refer to this <a href="http://www.webmonkey.com/2010/02/color_charts/">handy list of colors</a>.').html_safe %>
+  </p>
+  <dl>
+    <dt>
+      <%= f.label :background_color, ts('Background color') %> 
     </dt>
-    <dd><%= f.text_field :headercolor %></dd>
-    
-    <dt><%= f.label :accent_color, ts('Accent color') %>
-    <%= link_to_help('skins-wizard-accent-color') %>
+    <dd>
+      <%= f.text_field :background_color, "aria-describedby" => "background-color-field-notes" %>
+      <p class="footnote" id="background-color-field-notes">
+        <%= ts("Name or hex code. Default: <code>#fff</code>".html_safe) %>
+      </p>
+    </dd>
+
+    <dt>
+      <%= f.label :foreground_color, ts('Text color') %>
     </dt>
-    <dd><%= f.text_field :accent_color %></dd>
-  </dl>
+    <dd>
+      <%= f.text_field :foreground_color, "aria-describedby" => "foreground-color-field-notes" %>
+      <p class="footnote" id="foreground-color-field-notes">
+        <%= ts("Name or hex code. Default: <code>#2a2a2a</code>".html_safe) %>
+      </p>
+    </dd>
 
-    <%= hidden_field_tag 'wizard', true %>
+    <dt>
+      <%= f.label :headercolor, ts('Header color') %> 
+    </dt>
+    <dd>
+      <%= f.text_field :headercolor, "aria-describedby" => "header-color-field-notes" %>
+      <p class="footnote" id="header-color-field-notes">
+        <%= ts("Name or hex code. Default: <code>#900</code>".html_safe) %>
+      </p>
+    </dd>
+
+    <dt>
+      <%= f.label :accent_color, ts('Accent color') %>
+    </dt>
+    <dd>
+      <%= f.text_field :accent_color, "aria-describedby" => "accent-color-field-notes" %>
+      <p class="footnote" id="accent-color-field-notes">
+        <%= ts("Name or hex code. Default: <code>#fff</code>".html_safe) %>
```

Comment:
I'm game for detailing what the colors are for, although I think I'll put it in as help text, since the explanations might get a little too long. (Also, note to self: the default here is wrong. This is a grey, despite what the old help text said.)